OsuPlayer.Wpf/Windows/MiniWindow.xaml.cs

```diff
@@ -26,7 +26,7 @@ public partial class MiniWindow : WindowEx
         private double _screenWidth;
         private double _screenHeight;
 
-        private Rectangle _currentArea;
+        private static Rectangle _currentArea;
```

Comment:
`static` shouldn't be here. The default value can be initialized from `Loaded` event

Reply:
That's my fault, fixed 5c1f431.